.travis.yml

```
@@ -1,15 +1,17 @@
 ---
+after_script:
+  - workflow/on-failure.sh
 before_install:
-  - 'sudo apt-get -qq update'
-  - 'pip install -r requirements.txt'
+  - sudo apt-get -qq update
```

Did we want to apply our platform norm of exploding out short flags in scripts here?  EG

```suggestion
  - sudo apt-get --quiet --quiet update
```